src/tiled/preferences.cpp

```diff
@@ -566,6 +567,14 @@ void Preferences::setPluginEnabled(const QString &fileName, bool enabled)
     mSettings->setValue(QLatin1String("Plugins/Enabled"), enabledPlugins);
 }
 
+void Preferences::setZoomPref(bool mode){
+    mZoomPref = mode;
```

Comment:
`mZoomPref`? Please use a more descriptive name like `mWheelZoomsByDefault`.

Please make sure the setting is persistent (see other settings for how to interact with the `mSettings` member).